StrictEmit/ILGeneratorExtensions.Utility.cs

```diff
@@ -30,46 +30,61 @@ public static partial class ILGeneratorExtensions
         /// </summary>
         /// <param name="il">The generator where the IL is to be emitted.</param>
         [PublicAPI]
-        public static void EmitBreakpoint([NotNull] this ILGenerator il) => il.Emit(OpCodes.Break);
+        public static void EmitBreakpoint([NotNull] this ILGenerator il)
+            => il.Emit(OpCodes.Break);
 
         /// <summary>
         /// Fills space if opcodes are patched. No meaningful operation is performed although a processing cycle can be
         /// consumed.
         /// </summary>
         /// <param name="il">The generator where the IL is to be emitted.</param>
         [PublicAPI]
-        public static void EmitNoOperation([NotNull] this ILGenerator il) => il.Emit(OpCodes.Nop);
+        public static void EmitNoOperation([NotNull] this ILGenerator il)
+            => il.Emit(OpCodes.Nop);
 
         /// <summary>
         /// Removes the value currently on top of the evaluation stack.
         /// </summary>
         /// <param name="il">The generator where the IL is to be emitted.</param>
         [PublicAPI]
-        public static void EmitPop([NotNull] this ILGenerator il) => il.Emit(OpCodes.Pop);
+        public static void EmitPop([NotNull] this ILGenerator il)
+            => il.Emit(OpCodes.Pop);
 
         /// <summary>
         /// Pushes the size, in bytes, of a supplied value type onto the evaluation stack.
         /// </summary>
-        /// <typeparam name="T">The type.</typeparam>
+        /// <typeparam name="T">The type to emit the size of in bytes.</typeparam>
```

Comment:
Also a little clunky. Perhaps "to calculate the size of"? "in bytes" is not required, we specify it above. Same for the others.